# request

Task[] tasks # each task has a list of action cores, each action core contains a list of action roles

---

# response

int32 status              # 0 if everything was ok, -1 otherwise
int32[] individual_status # for each task, 0 if everything was ok, -1 otherwise
string[] messages         # for each task, in case of error: message what went wrong
string[] plan_strings     # for each task, something to describe the plan call

